cloudnetpy/plotting/plotting.py

```diff
@@ -328,7 +407,7 @@ def set_ax(ax, max_y: float, ylabel: str | None, min_y: float = 0.0):
 def _get_standard_time_ticks(resolution: int = 4) -> list:
     """Returns typical ticks / labels for a time vector between 0-24h."""
     return [
-        f"{int(i):02d}:00" if 24 > i > 0 else ""
+        f"{int(i):02d}:00" if 24 >= i >= 0 else ""
```

Comment:
I was considering this but wanted to avoid adding more and more keywords to generate_figure as at some point nobody will use them anymore. Personally, I am always suspicious if a figure does not have the limits labelled but can switch it to be a kw.